specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/examples/TrainBatch.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "body": {},
+    "trainRequest": {
+      "source": "/input/data1"
```

Comment:
Should we provide a sample that indicates Azure Blob Uri? This sample will only work on self-hosted containers.

Reply:
done